javaparser-testing/src/test/java/com/github/javaparser/printer/lexicalpreservation/transformations/ast/body/FieldDeclarationTransformationsTest.java

```diff
@@ -44,24 +44,33 @@ protected FieldDeclaration consider(String code) {
     // Modifiers
 
     @Test
-    public void addingModifiers() throws IOException {
+    public void addingModifiers() {
         FieldDeclaration it = consider("int A;");
         it.setModifiers(EnumSet.of(Modifier.PUBLIC));
         assertTransformedToString("public int A;", it);
     }
 
     @Test
-    public void removingModifiers() throws IOException {
+    public void removingModifiers() {
         FieldDeclaration it = consider("public int A;");
         it.setModifiers(EnumSet.noneOf(Modifier.class));
         assertTransformedToString("int A;", it);
     }
 
     @Test
-    public void replacingModifiers() throws IOException {
+    public void replacingModifiers() {
         FieldDeclaration it = consider("int A;");
         it.setModifiers(EnumSet.of(Modifier.PROTECTED));
         assertTransformedToString("protected int A;", it);
     }
 
+    @Test
+    public void changingTypes() {
+        FieldDeclaration it = consider("int a, b;");
+        assertTransformedToString("int a, b;", it);
+        it.getVariable(0).setType("Xyz");
+        assertTransformedToString(" a, b;", it);
```

Comment:
Lex pres printer prints nothing when the type is inconsistent, and I don't know how to make it print something - I guess it's not really how it's supposed to work. Alternatively, we could throw an exception here. What do you think @ftomassetti ?

Reply:
I would say that throwing an exception makes sense. This is better than producing wrong code. Optionally we could make this configurable, but I would go for the exception

Reply:
I'm not sure how I would throw an exception for this - isn't this all the way in the CSM where we're printing an IDENTIFIER which is null?